### PR TITLE
Fix Dock

### PR DESCRIPTION
**Pull request by flamendless**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Pre-built binaries for Windows and Mas OSX are provided in the [releases](https:
 
 Simple integration:
 ```lua
-require "imgui"
+local imgui = require "imgui"
```
Comment (MikuAuahDark) [inline README.md:40]: Good thing you noticed this part.

File: src/imgui_iterator_dock.h [hunk 2]
```diff
@@ -24,7 +24,8 @@ DEFAULT_ARG(bool*, opened, NULL)
 OPTIONAL_ENUM_ARG(extra_flags, 0)
 CALL_FUNCTION(BeginDock, bool, label, opened, extra_flags)
 IF_RET_ADD_END_STACK(1)
-PUSH_BOOL(ret)
+PUSH_BOOL(ret
```
Comment (MikuAuahDark) [inline src/imgui_iterator_dock.h]: ???

**Review by MikuAuahDark — CHANGES_REQUESTED** [review]
The `imgui_iterator_dock.h` contains typo where clang complains.

File: src/imgui_iterator_dock.h (hunk 2 above)
Comment (MikuAuahDark) [reply src/imgui_iterator_dock.h]: This part causes clang to complain. Is this intended?

**Review by flamendless — COMMENTED** [review]
No more review error

**Review by MikuAuahDark — APPROVED** [review]
Alright clang no longer complains.

**Pull request merged**